pkg/flags/parse.go

```diff
@@ -23,12 +39,17 @@ func (f Flags) ConnString() string {
 	if f.sslmode {
 		sslmode = "require"
 	}
+
 	flagStr := fmt.Sprintf("user=%s dbname=%s sslmode=%s", f.user, f.dbname, sslmode)
+	if f.dbpass != "" {
+		flagStr += fmt.Sprintf(" password='%s'", f.dbpass)
+	}
 	return flagStr
 }
 
 func Parse() Flags {
 	user := flag.String("user", "", "username of postgres db")
+	askForPass := flag.Bool("W", false, "ask for password")
```

Comment:
is `W` conventional for this type of flag? also any reason it is capitalized? how about `-askpass`?